packages/lint/src/index.ts

```diff
@@ -0,0 +1,32 @@
+import { EsLinter, StyleLinter } from './linter';
+import type { ILintArgs, ILinterOpts } from './types';
+
+export type { ILintArgs, ILinterOpts };
+
+export default (opts: ILinterOpts, args: ILintArgs) => {
+  if (!args.eslintOnly) {
+    const stylelint = new StyleLinter(opts);
+    const STYLE_EXTS = ['**/*.js', '**/*.jsx', '**/*.ts', '**/*.tsx'];
+    if (!args.cssinjs) {
+      for (const suffix of STYLE_EXTS) {
+        args._.unshift('--ignore-pattern', suffix);
+      }
+    }
+    stylelint.run(args);
+  }
+
+  if (!args.stylelintOnly) {
+    const eslint = new EsLinter(opts);
+    const ES_EXTS = [
+      '**/*.less',
+      '**/*.css',
+      '**/*.sass',
+      '**/*.scss',
+      '**/*styl',
```

Comment:
```suggestion
      '**/*.styl',
```

这两个常量提到这个文件的顶层， 插在第 3 行；以及常量名写反了，`STYLE_ETXS` 代表样式相关的后缀名